system-int-test/src/cmr/system_int_test/data2/granule_counts.clj

```diff
@@ -77,7 +77,43 @@
   (into {} (for [{:keys [concept-id has-granules]} (:items results)]
              [concept-id has-granules])))
 
+(defmethod results->actual-has-granules :iso19115
+  [result-format results]
+  (into {} (for [{:keys [concept-id has-granules]} (:items results)]
+             [concept-id has-granules])))
+
```

Comment:
Didn't add defmethod for :umm_json case because has-granules doesn't exist in the umm_json result and it is not in the scope of this ticket - which is about granule_count.